Automattic/jetpack
# Adds a conditional themes block to the Plans tab.

#### Changes proposed in this Pull Request:
* Adds a new REST endpoint that queries WordPress.com about additional features that are available if you upgrade to a higher tier.
* Adds the data from that REST endpoint to redux state.
* Adds a block in the Plans page that shows a themes block if WordPress.com returns the themes feature slug among the available features.

![screenshot from 2017-06-23 00-50-38](https://user-images.githubusercontent.com/374293/27457712-c4b9cb96-57b6-11e7-924a-3b0add64b3ed.png)

#### Testing instructions:
* You will need D6098 in order for things to work properly. Also you will need to set `define( 'JETPACK__WPCOM_JSON_API_HOST', 'your.dotcom.sandbox' );'
* Make sure the `premium-themes` slug is returned as one of the keys in the `available` property of the `/sites/%s/features` endpoint.
* Open the Plans page, you should see the themes block at the top.
* The button should lead you to the plans compare page with proper tracking info.
